src/pipeline.cc

```diff
@@ -809,6 +809,9 @@ class PipelineWorker : public Nan::AsyncWorker {
             ->set("squash", baton->tiffSquash)
             ->set("compression", baton->tiffCompression)
             ->set("predictor", baton->tiffPredictor)));
+            ->set("predictor", baton->tiffPredictor)
```

Comment:
The position of the brackets are fixed by the way. But the dpi on the buffer is still not working.